LiveChat/index.js

```diff
@@ -370,6 +375,9 @@ export default class LiveChat extends Component {
 
 			customerSDK.listChats().then((data) => {
 				const { chatsSummary, totalChats } = data
+				if (!totalChats) {
+					this.removePlaceHolder();
+				}
```

Comment:
```suggestion
					this.removePlaceHolder();
```
At this point we know we got the chat data, so in any case, we need to remove the placeholder regardless  `totalChats`